### DISCUSSION TITLE
Function that allows to hide in explorations all columns that have no data recorded

### DISCUSSION THREAD

**Discussion by jeromeperez** [mathesar-foundation/mathesar#3446] [Feature Ideas]
It would be interesting to have a function that would allow you to hide in explorations all columns that have no data recorded or a particular value.
For example:
- where all records have a null value
- with all records equal to 0
- with all records equal to 'don't know
- etc ...

**Comment by seancolsen** [comment]
Thanks for this idea, @jeromeperez
